std_msgs/Header header

uint32 height
uint32 width

bool is_bigendian

uint32 point_step

uint32 row_step

uint8[] compressed_data

bool is_dense
uint32  size
